Plots/XY/NCL_xy_18.py

```diff
@@ -0,0 +1,124 @@
+vds = xr.open_mfdataset(vfiles, concat_dim='case', combine='nested',
+                        preprocess=assume_noleap_calendar, decode_times=False)
+
+gds = xr.open_dataset("../../data/netcdf_files/gw.nc")
+gds = gds.expand_dims(dim={'lon': nds.lon})
+
+###############################################################################
+# Some parameters
+nyrs = len(nds["time"])
+nlon = len(nds["lon"])
+nlat = len(nds["lat"])
+time =
+
+###############################################################################
+# OBS
+
+obs = np.loadtxt("../../data/ascii_files/jones_glob_ann_2002.asc", dtype=float)
+
+###############################################################################
+# NCL-based Weighted Mean Function
+
+def horizontal_weighted_mean(var, wgts):
+    return (var * wgts).sum(dim=['lat', 'lon']) / wgts.sum(dim=['lat', 'lon'])
+
+###############################################################################
+# NATURAL
+
+gavn = horizontal_weighted_mean(nds["TREFHT"], gds["gw"])
+gavan = gavn - gavn.isel(time=slice(0,30)).mean(dim='time')
+
+###############################################################################
+# ALL
+
+gavv = horizontal_weighted_mean(vds["TREFHT"], gds["gw"])
+gavav = gavv - gavv.isel(time=slice(0,30)).mean(dim='time')
+
+###############################################################################
+# CALCULATE ENSEMBLE MIN & MAX
+
+gavan_min = gavan.min(dim='case')
+gavan_max = gavan.max(dim='case')
+gavan_avg = gavan.mean(dim='case')
+
+gavav_min = gavav.min(dim='case')
+gavav_max = gavav.max(dim='case')
+gavav_avg = gavav.mean(dim='case')
+
+obs_avg = obs[34:144] - np.mean(obs[34:64])
+
+###############################################################################
+# Create plot
+
+fig, ax = plt.subplots(figsize=(10.5, 6))
+
+ax.tick_params(labelsize="small")
+ax.minorticks_on()
+ax.xaxis.set_minor_locator(tic.AutoMinorLocator(n=4))
+ax.yaxis.set_minor_locator(tic.AutoMinorLocator(n=3))
+ax.tick_params(axis="both", labelsize=20)
+ax.tick_params("both", length=8, width=1.50, which="major", bottom=True, top=True, left=True, right=True)
+ax.tick_params("both", length=5, width=0.75, which="minor", bottom=True, top=True, left=True, right=True)
+
+time = [t.year for t in gavan.time.values]
+
+ax.set_title('Parallel Climate Model Ensembles', fontsize=24, pad=60.0)
+ax.text(0.5, 1.125, 'Global Temperature Anomalies', fontsize=18, ha='center', va='center', transform=ax.transAxes)
+ax.text(0.5, 1.06, 'from 1890-1919 average', fontsize=14, ha='center', va='center', transform=ax.transAxes)
+ax.set_ylabel('$^\circ$C', fontsize=24)
+ax.fill_between(time, gavan_min, gavan_max, color='lightblue', zorder=0)
+ax.fill_between(time, gavav_min, gavav_max, color='lightpink', zorder=1)
+
+ax.set_xlim(xmin=1890, xmax=2000)
+ax.set_ylim(ymin=-0.4, ymax=1)
+ax.set_xticks(np.arange(1900, 2001, step=20))
+ax.set_yticks(np.arange(-0.3, 1, step=0.3))
+
+ax.plot(time, obs_avg, color='black', label='Observations', zorder=4)
```

Comment:
`obs_avg.plot(x="time", color=...)` ?

Reply:
It is a Numpy array, not an Xarray `DataArray`.

Reply:
I tried, but it doesn't work.  It seems to me that the time axis, because it uses a `CFTimeIndex`, cannot be correctly used with the axis `Locator` object.  In other words, I can't set the axis major ticks as needed.  So, I'm going to keep the original implementation.